<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>interfacelogin</class>
 <widget class="QMainWindow" name="interfacelogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1411</width>
    <height>826</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>110</y>
      <width>571</width>
      <height>551</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame
{
border-radius:20px;
border:solid;
border-color:white;
padding:5px;
color:rgb(19,65,43);
background-color:rgb(225,209,182);
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="LineEditEmai">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>210</y>
       <width>411</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit
{
border:solid;
border-color:rgb(28,96,101);
border-width:1px;
background-color:rgb(255,255,255);
padding-left:15px;
border-radius:15px;
height:35px;
}</string>
     </property>
     <property name="placeholderText">
      <string>ðŸ“§</string>
     </property>
    </widget>
    <widget class="QPushButton" name="LoginButton">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>430</y>
       <width>191</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
border-radius:15px;
margin-top:15px;
height:40px;
background-color:rgb(97,125,77);
color: white;
font-weight:600;
}
QPushButton:hover
{
background-color:rgb(47,89,73);
}</string>
     </property>
     <property name="text">
      <string>Connexion</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>430</y>
       <width>211</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:15px;
margin-top:15px;
height:40px;
 background: rgb(187,135,126);
color: white;
font-weight:600;</string>
     </property>
     <property name="text">
      <string>CrÃ©er un nouveau Compte</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="LineEditMDP">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>320</y>
       <width>411</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit
{
border:solid;
border-color:rgb(28,96,101);
border-width:1px;
background-color:rgb(255,255,255);
padding-left:15px;
border-radius:15px;
height:35px;
}</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>ðŸ”‘</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_84">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>150</y>
       <width>71</width>
       <height>33</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(19,65,43);
font-weight:600;

</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Email&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_85">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>270</y>
       <width>121</width>
       <height>33</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(19,65,43);
font-weight:600;

</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Mot de Passe&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="NomEntreprise">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>30</y>
       <width>211</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 30pt &quot;Freestyle Script&quot;;
color:rgb(36,122,75);</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:36pt; font-weight:700;&quot;&gt;Purestay&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>0</y>
       <width>141</width>
       <height>141</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>351</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="purestaybackground.qrc">:/Purestay/images/interface/Logo PureStay Final.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>510</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Mot de passe oubliÃ©?</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>400</y>
       <width>191</width>
       <height>26</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(225,209,182);
color:rgb(19,65,43);
font-weight:600;
padding:5px;</string>
     </property>
     <property name="text">
      <string>Se Souvenir de moi</string>
     </property>
    </widget>
    <widget class="QLabel" name="LabelEmail">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>150</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(214,36,84);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="LabelMDP">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>270</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(214,36,84);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="LabelLogin">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>400</y>
       <width>171</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(214,36,84);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>LineEditEmai</zorder>
    <zorder>LoginButton</zorder>
    <zorder>pushButton_3</zorder>
    <zorder>LineEditMDP</zorder>
    <zorder>label_84</zorder>
    <zorder>label_85</zorder>
    <zorder>label_5</zorder>
    <zorder>label_6</zorder>
    <zorder>checkBox</zorder>
    <zorder>NomEntreprise</zorder>
    <zorder>LabelEmail</zorder>
    <zorder>LabelMDP</zorder>
    <zorder>LabelLogin</zorder>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1401</width>
      <height>761</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame
{
border-image: url(:/Purestay/images/interface/backgroundlogin2.png);
background-size:cover;
}

</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <zorder>frame_2</zorder>
   <zorder>frame</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1411</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="purestaybackground.qrc"/>
 </resources>
 <connections/>
</ui>
